webapp/src/components/collapsible_checklist.tsx

```diff
@@ -116,7 +116,17 @@ const ProgressLine = styled.div<{ width: number }>`
     }
 `;
 
+const HorizontalBG = styled.div`
+    background-color: var(--center-channel-bg);
+    z-index: 1;
+    position: sticky;
+    top: 48px; // height of rhs_checklists MainTitle
+`;
+
 const Horizontal = styled.div`
+    background-color: var(--center-channel-color-04);
+    border-radius: 4px 4px 0 0;
+    border: 1px solid var(--center-channel-color-08);
```

Comment:
Should we try to always use the rgba ones?
```suggestion
    background-color: rgba(var(--center-channel-color-rgb, 0.04));
    border-radius: 4px 4px 0 0;
    border: 1px solid rgba(var(--center-channel-color-rgb, 0.08));
```